canonical-web-and-design/certification.ubuntu.com
# Adds basic flask application setup

# Done
- copied the base flask application from https://github.com/canonical-web-and-design/canonical.com

Fixes https://github.com/canonical-web-and-design/base-squad/issues/587

# QA
`-./run`
- head to `http://localhost:8034/test` and make sure that the markdown renders correctly
